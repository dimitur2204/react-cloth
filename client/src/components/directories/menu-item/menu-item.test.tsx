import { shallow, ShallowWrapper } from 'enzyme';
import { BrowserRouter } from 'react-router-dom';
import MenuItem from './menu-item.component';

describe('MenuItem', () => {
	let wrapper: ShallowWrapper;
	const imageUrl = 'testImageUrl';
	const linkUrl = 'testLinkUrl';
	const title = 'testTitle';
	const size = 'testSize';

	beforeEach(() => {
		wrapper = shallow(
			<BrowserRouter>
				<MenuItem
					title={title}
					linkUrl={linkUrl}
					imageUrl={imageUrl}
					size={size}
				/>
			</BrowserRouter>
		);
	});

	it('should render MenuItem component', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
