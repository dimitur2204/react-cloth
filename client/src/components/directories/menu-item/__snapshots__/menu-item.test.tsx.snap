// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuItem should render MenuItem component 1`] = `ShallowWrapper {}`;
